src/scripts/towns/Town.ts

```diff
@@ -995,15 +1060,24 @@ const SinnohBerryMaster = new Shop([
 ]);
 
 //Sinnoh NPCs
+
+const TwinleafContestChampion = new NPC('Contest Champion', [
+    'Welcome to Sinnoh! There are many legends and myths here. For example, it is said that trainers that conquer the Lake nearby, Lake Verity, will meet a mythical Pokémon known for Emotion roaming around the region. It sure would have been awesome to partner with that Pokémon in one of my routines!',
```

Comment:
Do we want to keep the distinction between legendary and mythical? Mythical being an event-only legendary Pokémon. By that definition Mesprit is not a mythical.